tests/end_to_end/netsh_test.cpp

```diff
@@ -250,15 +259,24 @@ TEST_CASE("show sections map_reuse_um.dll", "[netsh][sections]")
     int result;
     std::string output = _run_netsh_command(handle_ebpf_show_sections, L"map_reuse_um.dll", nullptr, nullptr, &result);
     REQUIRE(result == NO_ERROR);
+
+#if defined(_M_X64) && defined(NDEBUG)
```

Comment:
[nit]:Add comments to explain the purpose of each block, especially if the code_size values have specific meanings or are derived from particular requirements for each architecture and mode.

This comment applies to each of the below  netsh command too!

Reply:
These are merely the sizes of the code generated by the compiler. They have no meaning beyond that.